OSSystems/meta-browser
# chromium: update to 83.0.4103.61

Stable release announcements since our latest update:
https://chromereleases.googleblog.com/2020/05/stable-channel-update-for-desktop_19.html

Test-built with:

License updates:

Updated licences:
* third_party/android_deps/libs/com_google_android_gms_play_services_*
  - GMSCore: Rolling Google Play Services 15.0.1 -> 17.0.0
  - https://crrev.com/c/2083221
* third_party/android_deps/libs/com_google_protobuf_protobuf_javalite/LICENSE
  - Roll protobuf javalite runtime + fix gn package name
  - https://crrev.com/c/1762684
* third_party/ffmpeg/CREDITS.chromium
  - M82 FFMpeg configuration updat
  - https://crrev.com/c/206563
* third_party/perfetto/NOTICE -> */LICENSE
  - Updated perfetto
  - https://crrev.com/c/2134619

Added licences:
* third_party/android_provider/LICENSE
  - Upstream internal DownloadCollectionBridge code
  - https://crrev.com/c/2090011
* third_party/pywebsocket3/src/LICENSE
  - Added pywebsocket3 to third_party
  - https://crrev.com/c/2059041
* third_party/schema_org/LICENSE
  - Add Schema.org schema file for eventual use in document metadata extractor
  - https://crrev.com/c/2023864
* third_party/weston/LICENSE
  - Add Weston to third_party.
  - https://crrev.com/c/1962128

Removed licenses:
* third_party/gestures/LICENSE && third_party/libevdev/LICENSE
  - Removed gestures and libevdev from third_party
  - https://crrev.com/c/2048906

* chromium-x11
  * zeus, MACHINE="intel-corei7-64"
  * zeus, MACHINE="intel-core2-32"
  * warrior, MACHINE="intel-corei7-64"
* chromium-ozone-wayland:
  * zeus, MACHINE="qemuarm64"
  * zeus, MACHINE="m3ulcb"
  * warrior, MACHINE="raspberrypi3"

Fixes #377
Signed-off-by: Maksim Sisov <msisov@igalia.com>
